cfme/tests/configure/test_logs.py

```diff
@@ -49,16 +50,16 @@ def test_provider_log_rotate(appliance, provider, log_exists):
     Metadata:
         test_flag: log
     """
-    if log_exists:
-        appliance.ssh_client.run_command("logrotate -f /etc/logrotate.d/miq_logs.conf")
-        logs_count = appliance.ssh_client.run_command(
-            "ls -l /var/www/miq/vmdb/log/{}.log*|wc -l".format(
-                provider.log_name
-            )
-        ).output
-        assert logs_count > 1, "{}.log wasn't rotated by default miq_logs.conf".format(
+    assert log_exists, "Log file {}.log doesn't exist".format(provider.log_name)
+    appliance.ssh_client.run_command("logrotate -f /etc/logrotate.d/miq_logs.conf")
+    logs_count = appliance.ssh_client.run_command(
+        "ls -l /var/www/miq/vmdb/log/{}.log*|wc -l".format(
             provider.log_name
         )
+    ).output
+    assert logs_count > 1, "{}.log wasn't rotated by default miq_logs.conf".format(
```

Comment:
I believe you'll need to explicitly cast this to an `int()`, as the output from SSH call will be a string. Trying to compare string and int with `>` results in TypeError